src/ApiPort.VisualStudio/Utils/TargetPlatformDisplayNameConverter.cs

```diff
@@ -20,7 +20,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
             if (target.AlternativeNames.Count > 0)
             {
-                return $"{target.Name} ({string.Join(", ", target.AlternativeNames)})";
+                return FormattableString.Invariant($"{target.Name} ({string.Join(", ", target.AlternativeNames)})");
```

Comment:
you can always do `using static System.FormattableString` then just call `Invariant($"...")` - not a big deal, though